src/model/liquid.rs

```diff
@@ -34,18 +34,22 @@ pub fn render_liquid<W: Write>(
         )
     };
 
+    let (cr, cg, cb) = if lava {
+        (255, 255, 255)
+    } else {
+        // TODO: Choose water color based on biome
+        (63, 118, 228)
+    };
+
     let tex = match snapshot.get_block(x, y, z) {
         block::Block::Water { .. } => {
-            render::Renderer::get_texture(&textures, "minecraft:blocks/water_still")
-        }
-        block::Block::FlowingWater { .. } => {
-            render::Renderer::get_texture(&textures, "minecraft:blocks/water_flow")
+            render::Renderer::get_texture(&textures, "minecraft:block/water_still")
```

Comment:
does this still work for older textures? or is that smth we should add in the future (support for older textures)